extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -17,9 +17,23 @@
  * @param {Object} [config] Configuration options
  */
 ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage( parameter, name, config ) {
+	var paramName = parameter.getName(),
+		paramType;
+
 	// Parent constructor
 	ve.ui.WikiaParameterPage.super.call( this, parameter, name, config );
 
+	paramType = this.spec.params && this.spec.params[paramName] && this.spec.params[paramName].type;
+
+	if (paramType === 'image') {
+		// TODO display something nice
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)